enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java

```diff
@@ -136,11 +137,10 @@ private KernelTransaction getCurrentTx()
         return terminateTransactionsForValidUser( graph.getDependencyResolver(), username, getCurrentTx() );
     }
 
+    //@Admin
```

Comment:
We took away the assert admin, so how is this procedure an admin procedure now?

Reply:
If you look at the row below, this procedure has been disabled from the surface.